website/header.shtml

```diff
@@ -29,6 +29,9 @@
     <li class="nav-item"><a
       class="nav-link<!--#if expr="v('DOCUMENT_NAME') = 'about.shtml'" --> active<!--#endif-->"
       href="about"><i class="far fa-lightbulb"></i> About</a></li>
+    <li class="nav-item"><a
+      class="nav-link<!--#if expr="v('DOCUMENT_NAME') = 'slack.shtml'" --> active<!--#endif-->"
+      href="slack"><i class="fab fa-slack"></i> Slack</a></li>
```

Comment:
would call it 'chat' or something more generic than the brand of chat we use. can keep the icon as a hint to which brand we're using today.

Reply:
👍 . Also if we upgrade font-awesome we get the new Slack icon

Reply:
Ah the upgrading needs to be done in demoweb